lib/authorized_persona/authorization.rb

```diff
@@ -77,16 +77,16 @@ def authorized?
     private
 
     def authorization_current_user
-      unless authorization_current_user_method.is_a?(Symbol)
-        raise AuthorizedPersona::Error, "you must configure authorization with a valid current_user method name, " \
-                                        "e.g. `authorize_persona class_name: 'User', current_user_method: :my_custom_current_user`"
+      unless authorization_current_user_method.instance_of?(Symbol)
+        raise Error, "you must configure authorization with a valid current_user method name, " \
+                     "e.g. `authorize_persona class_name: 'User', current_user_method: :my_custom_current_user`"
       end
 
-      send(self.class.authorization_current_user_method)
+      __send__(authorization_current_user_method)
```

Comment:
Considered `public_send` for this, but decided that `current_user` should be allowed to be private (added test coverage for that).